src/app/models/types/global.ts

```diff
@@ -1,3 +1,7 @@
+export type AppEnvs = 'local' | 'test' | 'live';
+
+export type AppPlatforms = 'amp' | 'canonical';
```

Comment:
Can we maybe rename these?
```suggestion
export type Environments = 'local' | 'test' | 'live';

export type Platforms = 'amp' | 'canonical';
```